wasm/actions/empty.go

```diff
@@ -0,0 +1 @@
+package actions
```

Comment:
add a comment explaining we need this otherwise go build isn't happy that we have a package that excludes things sometimes all of the time:#